scripts/c2r_script.py

```diff
@@ -685,38 +750,48 @@ def main():
     finally:
         # Gather JSON & Textual report
         data = gather_json_report()
-        if data and not rollback_errors:
+
+        if data:
             output.status = data.get("status", None)
 
             # At this point we know JSON report exists and no rollback errors occured
             # we can rewrite possible previous message with more specific one and set alert
-            output.message, output.alert = generate_report_message(output.status)
+            if not rollback_errors and (
+                SCRIPT_TYPE == "CONVERSION" or not output.message
+            ):
```

Comment:
:thinking: probably rename this, it's basically case of either no rollback errors in conversion or in analysis without message